swagger/swagger.yaml

```diff
@@ -578,22 +578,32 @@ paths:
           description: Comma-separated list of field names to retrieve for directories
           required: false
           type: string
+        - name: file_name
+          in: query
+          description: Substring search from file names. Only matching files are returned.
+          required: false
+          type: string
+        - name: directory_name
+          in: query
+          description: Substring search from directory names. Only matching directories are returned.
+          required: false
+          type: string
```

Comment:
Both parameters could include a note about search scope, which I think is the current directory?